### PR TITLE
Minor update on SDK verbiage

### PR DESCRIPTION
**Pull request by animeshsingh**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/286)

### CODE REVIEW

File: python/kfserving/README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ KFServing supports the following storage providers:
 
 ### Getting Started
 
-KFServing's python client provides many APIs for operations of KFServing, such as creating, patching and deleting. See the [Sample for KFServing Python SDK Client](../../docs/samples/client/kfserving_sdk_sample.ipynb) to get started.
+KFServing's python client interacts with KFServing APIs for exeucting operations on a remote KFServing server instance, such as creating, patching and deleting of a KFService instance. See the [Sample for KFServing Python SDK Client](../../docs/samples/client/kfserving_sdk_sample.ipynb) to get started.
```
Comment (rakelkar) [inline python/kfserving/README.md]: typo
```suggestion
KFServing's python client interacts with KFServing APIs for executing operations on a remote KFServing server instance, such as creating, patching and deleting of a KFService instance. See the [Sample for KFServing Python SDK Client](../../docs/samples/client/kfserving_sdk_sample.ipynb) to get started.
```

Comment (rakelkar) [reply python/kfserving/README.md]: instead of remote server instance maybe say on a KFServing deployment or cluster?

Comment (animeshsingh) [reply python/kfserving/README.md]: thanks @rakelkar - had 'cluster' there before. Will update

Comment (animeshsingh) [reply python/kfserving/README.md]: Done

**Pull request merged**